src/components/Sections/SectionHeader.tsx

```diff
@@ -0,0 +1,50 @@
+import React from 'react'
+
+import ActionsMenu from 'cozy-ui/transpiled/react/ActionsMenu'
+import Button from 'cozy-ui/transpiled/react/Buttons'
+import Divider from 'cozy-ui/transpiled/react/MuiCozyTheme/Divider'
+import DotsIcon from 'cozy-ui/transpiled/react/Icons/Dots'
+import Icon from 'cozy-ui/transpiled/react/Icon'
+
+import { actions } from 'components/Sections/SectionActions'
+import { SectionHeaderProps } from 'components/Sections/SectionsTypes'
+
+export const SectionHeader = ({
+  name,
+  showMore,
+  anchorRef,
+  toggleMenu,
+  menuState,
+  handleAction
+}: SectionHeaderProps): JSX.Element => (
+  <>
+    <Divider className="u-mv-0" variant="subtitle2">
+      {name}
+    </Divider>
+
+    {showMore && (
```

Comment:
I think you should use the flag('') directly here so we can remove it after the flag is deploy and avoir missing it

Reply:
✅